asv_bench/benchmarks/io/csv.py

```diff
@@ -172,6 +172,26 @@ def time_head_of_multiindex(self):
         self.df_custom_index_then_head.to_csv(self.fname)
 
 
+class ToCSVSparse(BaseIO):
+
+    fname = "__test__.csv"
+
+    def setup(self):
+        from scipy import sparse as sc
+
+        vals = np.random.randint(0, 10, size=(10000, 1000))
+        keep = vals > 3
+        vals[keep] = 0
+        sparse_mtx = sc.coo_matrix(vals)
+        self.data = DataFrame.sparse.from_spmatrix(sparse_mtx)
+
+    def time_sparse_to_csv(self):
+        self.data.to_csv("sparse_pd.csv")
+
+    def time_sparse_to_dense_to_csv(self):
+        self.data.sparse.to_dense().to_csv("sparse_pd.csv")
```

Comment:
Can you also please shrink the size of the data for the benchmark? I think we usually target < 100ms, but based on PR description looks like this is much slower (especially before this commit)